app/views/docs/offline.phtml

```diff
@@ -4,17 +4,57 @@
 
 <h2>When Should I Enable Offline Support?</h2>
 <p>
-    With Offline Support enabled, your app can continue sending requests to Appwrite APIs while offline through a cache. While offline, read requests will be fetched from the local cached data, and write requests will be queued and executed once the device is online again.
+    You should consider enabling Offline Support if you expect the device running your application to experience short periods without a network connection, like going on an airplane or through a tunnel. Offline Support can't help you if the app's data needs to remain up to date with frequent updates or if write requests need to be processed immediately. If you expect your app to be offline for days or weeks at a time, you may also need to find alternative solutions.
 </p>
+
+<h2>Enable Offline Support</h2>
+
 <p>
-    You should consider enabling Offline Support if you expect the device running your application to experience short periods without a network connection, like going on an airplane or through a tunnel. Offline Support can't help you if the app's data needs to remain up to date with frequent updates or if write requests need to be processed immediately. If you expect your app to be offline for days or weeks at a time, you may also need to find alternative solutions.
+    Offline Support can be enabled by setting <code>setOffilnePersistency</code> to true on your SDK's client.
+</p>
+
+<div class="notice">
+  <h2>Adding Offline Support to Existing Projects</h2>
+  <p>Enabling Offline Support introduces breaking changes to the behavior of Client SDKs. Asynchronous operations may not resolve until the device is online again. Existing projects will have to be updated.</p>
+  <p><a href="#async-operation">Learn more about asynchronous operations</a></p>
+</div>
+
+[TODO: Code Example]
+<p>
+    Offline Support will cache all read request in a <a href="https://en.wikipedia.org/wiki/Cache_replacement_policies#LRU" target="_blank" rel="noopener">Least Recently Used (LRU)</a> cache, so the information can remain available even with no internet connection. The LRU cache defaults to a size of 16KB and can be configured using the <code>setOfflineCacheSize</code> method.
 </p>
 
-<h3>Considerations and Tradeoffs</h3>
+[TODO: Code Example]
+
+<h2 href="#async-operation" id="async-operation">Asynchronous Operation</h2>
 <p>
-    Offline Support adds a cache layer that takes up additional resources and adds overhead to your requests. This may not be desirable for some applications, especially if it's not expected to be used without a stable internet connection.
+    When Offline Support is enabled, asynchronous operations only resolve after the request is accepted by the server. Requests made with the Client SDK may block code execution until network connection is available again.
```

Comment:
Let's clarify this as asynchronous write operations. And also a Grammarly suggestion.

```suggestion
    When Offline Support is enabled, asynchronous write operations only resolve after the server accepts the request. Requests made with the Client SDK may block code execution until network connection is available again.
```